doc/releases/release-notes-4.0.rst

```diff
@@ -145,13 +145,17 @@ Boards & SoC Support
 
 * Added support for these SoC series:
 
+  * Raspberry Pi RP2350
+
 * Made these changes in other SoC series:
 
   * NXP S32Z270: Added support for the new silicon cut version 2.0. Note that the previous
     versions (1.0 and 1.1) are no longer supported.
 
 * Added support for these boards:
 
+  * :ref:`Raspberry Pi Pico 2 <rpi_pico2>`: ``rpi_pico2``.
```

Comment:
```suggestion
  * :zephyr:board:`Raspberry Pi Pico 2 <rpi_pico2>`: ``rpi_pico2``.
```

Reply:
Pulling in this change  has caused the documentation to fail to build, @kartben . See https://github.com/zephyrproject-rtos/zephyr/actions/runs/11524102239/job/32083564294#step:8:60  . I'll leave my branch as-is and broken for now, so you can see if I've done something wrong. If it's still broken on Monday Europe-time I'll revert this suggestion.

Reply:
All good now! Thanks much.

https://builds.zephyrproject.io/zephyr/pr/77368/docs/boards/raspberrypi/rpi_pico2/doc/index.html

Sorry that I didn't spot this cross reference when asking you to use the directive in the board's doc page.

Reply:
Ah, sorry, I got got my CI actions/runs muddled up. Yup it all looks good, thanks for providing the changes as suggestions, it really reduces burden on contributors/reviewees.